doc/source/whatsnew/v1.0.0.rst

```diff
@@ -235,6 +235,26 @@ The following methods now also correctly output values for unobserved categories
    df.groupby(["cat_1", "cat_2"], observed=False)["value"].count()
 
 
+By default :meth:`Categorical.min` and :meth:`Categorical.max` return the min and the max respectively
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When :class:`Categorical` contains ``np.nan``, :meth:`Categorical.min` and :meth:`Categorical.max`
+no longer return ``np.nan`` by default.
```

Comment:
Maybe add something like "to honor the default of `skipna=True`" to make it clear that this change makes it consistent with the rest of pandas